src/sentry/testutils/cases.py

```diff
@@ -1567,3 +1577,76 @@ def setUp(self):
         )
         self.name = self.user.get_display_name()
         self.short_id = self.group.qualified_short_id
+
+
+@apply_feature_flag_on_cls("organizations:metrics")
+@pytest.mark.usefixtures("reset_snuba")
```

Comment:
I don't think you need to include this, since this inherits from `SnubaTestCase` at some point, which autouses the `reset_snuba` fixture.

Reply:
Hmm yeah I saw that but i don't think that's working as we expect it to.. because if i comment out this line and run the tests consecutively, in the second run some of the tests fail, and I think it is probably because the dataset is not cleared properly 
example:

```
[6.0, 12.0, 18.0, 24.0] == [3, 6, 9, 12]
```

Reply:
👍  Yeah I'd like to understand why that might not be running, because we do rely on that running for other snuba tests. If it's broken it could explain some flakiness.